includes/admin/class-wc-rest-stripe-settings-controller.php

```diff
@@ -244,9 +244,9 @@ public function register_routes() {
 	 */
 	public function get_settings() {
 		$is_upe_enabled               = WC_Stripe_Feature_Flags::is_upe_checkout_enabled();
+		$enabled_payment_method_ids   = $is_upe_enabled ? $this->gateway->get_upe_enabled_payment_method_ids( true ) : WC_Stripe_Helper::get_legacy_enabled_payment_method_ids();
```

Comment:
It may be worth adding a comment around the choice to trigger these calls in this order, just to make it clear why we're making the calls with the `true` argument in only this first call.
```suggestion
		// Note that we want to refresh the Stripe Payment Method Configuration cache exactly once
		// when UPE is enabled, so we force a refresh on this first call only, and not when calling
		// get_upe_available_payment_methods() below.
		$enabled_payment_method_ids   = $is_upe_enabled ? $this->gateway->get_upe_enabled_payment_method_ids( true ) : WC_Stripe_Helper::get_legacy_enabled_payment_method_ids();
```

Reply:
💟